Libraries/acdriver/Sources/CompileActionImageSet.cpp

```diff
@@ -296,7 +297,7 @@ CompileAsset(
 
     if (FSUtil::IsFileExtension(filename, "png", true)) {
         std::vector<uint8_t> contents;
-        if (!catalog->filesystem() || !catalog->filesystem()->read(&contents, filename)) {
+        if (!filesystem || !filesystem->read(&contents, filename)) {
```

Comment:
Even though pointers can be `nullptr`, I wouldn't bother checking for it — there's clearly no meaning to this function without a value provided, and if there was one, it'd be an `ext::optional<Filesystem const *>` anyway.